Desktop/modules/dynamicmodules.cpp

```diff
@@ -167,8 +168,11 @@ bool DynamicModules::initializeModule(DynamicModule * module)
 			emit dynamicModuleChanged(module);
 			emit loadModuleTranslationFile(module);
 			restartEngines();
+
```

Comment:
Why this restartEngines does not have an emit. I know it does not do anything, but it's nice to see that you are using a signal.

Reply:
Yeah, I think they maybe weren't signals to begin with. I'll check em.